examples/build.gradle

```diff
@@ -1,18 +1,19 @@
-apply plugin: 'com.android.application'
+apply plugin: 'com.android.library'
```

Comment:
This change looks suspicious. The examples directory contains a standalone application, so I would not expect the plugin type to be changed to library.

Reply:
Ye,  you are right. That one might slipped me